CHANGELOG.md

```diff
@@ -29,6 +29,8 @@
 - ["Write" button in component menu allows to evaluate it separately from the
   rest of the workflow][11523].
 - [The documentation editor can now display tables][11564]
+- [Table Input Widget is now used on Table.input method instead of
+  Table.new][11612].
```

Comment:
I would explain more for users not accustomed to our libs. Maybe "Table widget is now matched for Table.input code, and the columns are parsed"...